pkg/data/run_store.go

```diff
@@ -30,7 +30,7 @@ type RunStore interface {
 		completed []simulator.CompletedMovement,
 		ignored []simulator.IgnoredMovement,
 		clusterConf model.ClusterConfig,
-		kpaConf model.KnativeAutoscalerConfig,
+		kpaConf model.AutoscalerConfig,
```

Comment:
All remaining stuff regarding Knative including renaming variables I fixed in another PR (Remove dependency on Knative). Once this PR is landed I'll do another one, because I need changes from this PR. But I can rename it in this PR if it does really matter.

Reply:
Fixed it.

Reply:
You fixed just one place, right? We should rename all 'kpa'.

But should be okay to do it in another PR dedicated to removing Knative.